### PR TITLE
Provide issue examples

### PR DESCRIPTION
**Pull request by cehsu**
## Description
These changes add examples to the GI Bill School Feedback Tool issues section.

## Screenshots
image

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Documentation has been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: src/applications/edu-benefits/feedback-tool/helpers.js
Comment (erikphansen) [inline src/applications/edu-benefits/feedback-tool/helpers.js]: 👏 Nicely done. I do wonder if there's more semantic markup to use here, but nothing comes to mind.

File: src/applications/edu-benefits/feedback-tool/helpers.js
Comment (erikphansen) [inline src/applications/edu-benefits/feedback-tool/helpers.js]: nit: I'd prefer to not have the extra line breaks between these. But I can live with this if others feel strongly.

**Review by erikphansen — APPROVED** [review]

File: src/applications/edu-benefits/feedback-tool/helpers.js
Comment (rianfowler) [reply src/applications/edu-benefits/feedback-tool/helpers.js]: Probably needs a tab index if we want screen readers to see these similar to the suggestion in [this](https://app.zenhub.com/workspace/o/department-of-veterans-affairs/vets.gov-team/issues/12670). defer to @1Copenut on this

Comment (cehsu) [reply src/applications/edu-benefits/feedback-tool/helpers.js]: Good point, @erikphansen, updated based on @1Copenut's [comment](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/12252#issuecomment-419987168) regarding how this can be structured.

**Review by rianfowler — APPROVED** [review]

**Review by peggygannon — CHANGES_REQUESTED** [review]
@cehsu I think since we're no longer making these a link, then we need to add content letting users know the text in parens are merely examples of what the feedback could be. (otherwise users might think their feedback has to be exactly what the example shows)

1. I'd like to propose the following content update to the intro sentence:
Which topic best describes your feedback? (Select all that apply.) 
Please note, below the topics are examples of what the feedback could include.

2.  for the Financial concern entry, please delete (for example, tuition or fee changes)

**Comment by cehsu** [comment]
Thanks @peggygannon, kindly refer to the updated screenshot:
image

**Comment by peggygannon** [comment]
Hi, @cehsu Thanks.. Can I see what the formatting would look like if we made the line look like this? 

**Please note:** Below the topics are examples of what the feedback could include.

**Comment by peggygannon** [comment]
@cehsu, OK, thank you for doing that. 

I think we should go with the first screenshot you showed, where the text is not bolded.

**Comment by cehsu** [comment]
Thanks @peggygannon sounds good. We will stick with the non-bold version.
Please note that this PR is blocked by a bug in the us-forms-system: https://github.com/usds/us-forms-system/issues/260.
Which is resulting in the description being displayed twice:
image

**Review by peggygannon — APPROVED** [review]

**Comment by cehsu** [comment]
Closing this for now, as this work is blocked.

**Pull request merged**
